content/en/blog/2022/announcing-community-manager.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Announcing the OpenTelemetry Community Manager
+linkTitle: Announcing OpenTelemetry Community Manager
+date: 2022-09-24
+author: Austin Parker
+---
+
+OpenTelemetry has demonstrated massive growth since its inception in 2019. What started as a handful of OpenTracing and OpenCensus maintainers and collaborators meeting at the Google campus and over Zoom has now grown into the second-most popular project in the CNCF behind Kubernetes itself. Over 5400 contributors and 700 companies have contributed code, issues, documentation, and invaluable feedback. 
```

Comment:
> Over 5400 contributors and 700 companies have contributed code, issues, documentation, and invaluable feedback. 

Can you link to the source of that data? I assume CNCF dev stats?

Reply:
Done